docs/en/Running-Hive-with-Alluxio.md

```diff
@@ -46,8 +46,8 @@ Hive's Hadoop jobs will store its intermediate results in Alluxio. Please follow
 
 ### Add additional Alluxio site properties to Hive
 
-If there are any Alluxio site properties you want to specify for Hive, add those to `hive-site.xml`. For example,
-change `alluxio.user.file.writetype.default` from default `MUST_CACHE` to `CACHE_THROUGH`:
+If there are any Alluxio site properties you want to specify for Hive, add those to `core-site.xml` to Hadoop configuration
+ directory on each node. For example, change `alluxio.user.file.writetype.default` from default `MUST_CACHE` to `CACHE_THROUGH`:
```

Comment:
I don't think that would work. Why would Hadoop understand alluxio's site properties? I think adding configs to `core-xml` is more appropriate because the idea is to append Alluxio config to Hadoop's config, and `core-site` is the place to specify config properties.

Moreover, `alluxio-site.properties ` contain both server and client configs. We should only need to add client-side properties to hadoop.